### PR TITLE
Update PhimpMeGallery.java

### PR DESCRIPTION
**Pull request by AliAyub007**
Correction of grammatical error in dialog ox after uploading at line 116.

Fixes issue #86 

Changes: Fixed Grammatical error in Dialog box after uploading photo. 

Screenshots for the change:

### CODE REVIEW

**Comment by pa1pal** [comment]
@AliAyub007 There should not be different PR for each line of change and that too the string is same in #99. Add the string in strings.xml resource and then use it here. 
Run the inspection in Android Studio and add other hardcoded strings as well.

**Comment by Abhi2424shek** [comment]
Pls inform when you are finished @AliAyub007, work with git on PC don't work in github on web

**Comment by AliAyub007** [comment]
@Abhi2424shek  I just did this and I want to pull request. Will it be a separate one or how do I make it with in this request #98? Can you please tell me I am new to all this.

**Comment by Abhi2424shek** [comment]
Well, use git on your PC. Commit to AliAyub007:patch-3 branch and the or will get updated.

**Comment by AliAyub007** [comment]
Thanks got it now.

**Comment by Abhi2424shek** [comment]
Is the grammar fixes done?

**Comment by AliAyub007** [comment]
Yes @Abhi2424shek

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -611,4 +611,5 @@
   <string name="delete_all_photos">Delete all Photos</string>
   <string name="hiede_show_photos">Hide or show local photos</string>
   <string name="show_local_photos">Show Local Photos</string>
+  <string name="Dialog_upload">This photo has been added to list upload photo!</string>
```
Comment (Abhi2424shek) [inline app/src/main/res/values/strings.xml]: Use small letters pls!

**Comment by Abhi2424shek** [comment]
Are you really sure, I can see more grammatical errors!

**Comment by AliAyub007** [comment]
Let me go through whole strings.xml I will get back in few minutes

**Comment by AliAyub007** [comment]
@Abhi2424shek  I have made changes if there is any left do let me know.

File: app/src/main/java/vn/mbm/phimp/me/PhimpMe.java [hunk 2]
```diff
@@ -822,6 +822,7 @@ public boolean onKeyDown(int keycode, KeyEvent event) {
                         .replace(R.id.fragment_container, frag)
                         .commit();
                 currentScreen = HomeScreenState.GALLERY;
+                keycode = 0;
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/PhimpMe.java]: As stated in issue we don't need this.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -591,24 +591,25 @@
...
-  <string name="ask_delete">Are you sure you want to delete this file ?</string>
-  <string name="no_accounts_connected">No Accounts Connected :(</string>
+  <string name="ask_delete">Are you sure you want to delete this file?</string>
+  <string name="no_accounts_connected">No Accounts Connected:(</string>
   <string name="max_file_size">Maximum File Size</string>
   <string name="delete_all_photos">Delete all Photos</string>
   <string name="hiede_show_photos">Hide or show local photos</string>
   <string name="show_local_photos">Show Local Photos</string>
+  <string name="dialog_upload">This photo has been added to list upload photo!</string>
 </resources>
```
Comment (Abhi2424shek) [inline app/src/main/res/values/strings.xml]: Add a new line at eof

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -612,4 +612,6 @@
   <string name="hiede_show_photos">Hide or show local photos</string>
   <string name="show_local_photos">Show Local Photos</string>
   <string name="dialog_upload">This photo has been added to list upload photo!</string>
+
+  
 </resources>
```
Comment (Abhi2424shek) [inline app/src/main/res/values/strings.xml]: The line has to be added after this tag

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -591,24 +591,27 @@
...
+  <string name="no_accounts_connected">No Accounts Connected:(</string>
   <string name="max_file_size">Maximum File Size</string>
   <string name="delete_all_photos">Delete all Photos</string>
   <string name="hiede_show_photos">Hide or show local photos</string>
   <string name="show_local_photos">Show Local Photos</string>
-</resources>
\ No newline at end of file
+  <string name="dialog_upload">This photo has been added to list upload photo!</string>
+
+
```
Comment (Abhi2424shek) [inline app/src/main/res/values/strings.xml]: Remove this line.

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -591,24 +591,27 @@
...
+  <string name="ask_delete">Are you sure you want to delete this file?</string>
+  <string name="no_accounts_connected">No Accounts Connected:(</string>
   <string name="max_file_size">Maximum File Size</string>
   <string name="delete_all_photos">Delete all Photos</string>
   <string name="hiede_show_photos">Hide or show local photos</string>
   <string name="show_local_photos">Show Local Photos</string>
-</resources>
\ No newline at end of file
+  <string name="dialog_upload">This photo has been added to list upload photo!</string>
+
```
Comment (Abhi2424shek) [inline app/src/main/res/values/strings.xml]: Remove this line

**Comment by Abhi2424shek** [comment]
Resolve the conflicts as well

**Pull request merged**